### DISCUSSION TITLE
Omada Controller as LXC container for Proxmox ? [Idea]:

### DISCUSSION THREAD

**Discussion by jonas740** [mbentley/docker-omada-controller#431] [Ideas]
### Describe the Feature/Enhancement/Idea

There might already be something like this but i am reffering to making an LXC container for Proxmox so that a simple script can install it.

Just like many other things found on this page [https://helper-scripts.com/](https://helper-scripts.com/)

If there is such a thing already then please share.

**Comment by mbentley** [comment]
It has been a really long time since I have looked at LXC personally or what it might require to get it working in a sort of setup for Proxmox w/LXC.  I am not sure what, if anything, would be transferable from the containerization of the app to LXC.  I would assume that it would be some combination in the commends from the [Dockerfile](https://github.com/mbentley/docker-omada-controller/blob/master/Dockerfile.v5.x) + the [install.sh](https://github.com/mbentley/docker-omada-controller/blob/master/install.sh) to get it at least installed though.  I don't have a Proxmox setup to do any sort of test myself at least.

**Reply by jonas740** [reply]
I just stumbled on this on YouTube https://youtu.be/e86G-B-nT6U?si=UvFDdcLeRFKs0sDl

That is why I though, hmmm, making a script to have that installed as a LXC container in proxmox would be quite sweet.

I use proxmox for a lot of other things.
